packages/terra-form/lib/Form.scss

```diff
@@ -0,0 +1,60 @@
+@import '~terra-mixins';
+
+.terra-Form-input,
+.terra-Form-textarea {
+  border: $terra-grey-15 solid 1px;
+  border-radius: 0.25em;
+  font-size: 16px;
```

Comment:
We do need to set the font-size, as the default font size is too small.

However, it does need to flex with the base font-size. Converted to em.

https://github.com/cerner/terra-core/pull/305/commits/ea8cbabd8d698303db9e476bc73d56dc79e9a75d